packages/core/src/controllers/SwapController.ts

```diff
@@ -443,21 +446,40 @@ export const SwapController = {
   },
 
   async getInitialGasPrice() {
-    const res = await SwapApiUtil.fetchGasPrice()
+    const res = await SwapApiUtil.fetchGasPrice().catch(() => null)
```

Comment:
why is this catch handled like this? shouldn't this be handled internally on `fetchGasPrice`  ?

Reply:
https://github.com/WalletConnect/web3modal/pull/2802/commits/624e690ff281d0defe108176d4adc0eb79521997